Jenkinsfile

```diff
@@ -48,6 +43,18 @@ node {
         stage("clang-debug") {
           sh "export CCACHE_BASEDIR=`pwd`; cd clang-debug && make all -j \$(( \$(cat /proc/cpuinfo | grep processor | wc -l) / 3))"
         }
+      }, moreLint: {
```

Comment:
Why do we split linting into two phases?

Reply:
Couldn't we just have the strict check only?

Reply:
See the discussion in #489. We don't want the CI build to fail if a test is disabled without an issue number.

During regular TDD, you will write tests that are not enabled yet. There is no issue number for them, because they are not an issue. Only when you merge them into the master, they should either be enabled or have a connected issue.

Reply:
I see. Nice.